lib/Experian/IDAuth.pm

```diff
@@ -86,11 +85,7 @@ sub save_pdf_result {
         # Download pdf result on given reference number
         $mech->get("$url/archive/index.cfm?event=archive.pdf&id=$our_ref");
         1;
-    } || do {
-        my $err = $@;
-        warn "errors downloading pdf: $err";
-        return;
-    };
+    } or die "errors downloading pdf: $@";
```

Comment:
If not using a `try`-like wrapper (e.g. `Syntax::Keyword::Try`) it's usually a good idea to

    local $@;

just before the `eval` block